index.html

```diff
@@ -8,6 +8,32 @@
   <title>Oscar's Portfolio</title>
 </head>
 <body>
+  <section id="pop-up" class="pop-up">
+    <div class="background-pop-up"></div>
+
+    <div class="pop-up-content">
+      <div class="pop-up-title">
+        <h2 id="pop-up-title"></h2>
+        <p id="close-window">X</p>
+      </div>
+
+      <ul class="pop-up-technologies" id="pop-up-technologies"></ul>
+
+      <div class="content-description">
+        <img src='' alt="pop-up-img" class="pop-up-img" id="pop-up-img">
+        <div class="description-text">
```

Comment:
- [ ]  Kindly stick as much as you can to the Figma design, please notice that the pop-up from your phone version has a side scroll on 375px, please remove it. 📈


https://user-images.githubusercontent.com/95243996/202276441-48ba29b7-0f1f-4958-bcbe-9695f22cc943.mp4